src/redux/epics/post-kb-credentials-user.js

```diff
@@ -23,8 +23,8 @@ export default ({ kbCredentialsUsersApi }) => (action$, state$) => {
         .assignUser(state$.value.okapi, credentialsId, userData)
         .pipe(
           map(postKBCredentialsUserSuccess),
-          map(() => getKBCredentialsUsers(credentialsId)),
-          catchError(errors => of(postKBCredentialsUserFailure({ errors }))),
+          map(() => getKBCredentialsUsers(userData.attributes)),
```

Comment:
This change breaks a request to get assigned users after successful assigning - `getKBCredentialsUser` action expects kb credentials string, that will be used to format request url, so we should revert this line to how it was

https://user-images.githubusercontent.com/19309423/176380137-bf8288b4-61a3-4755-8358-d3c708728bf4.mp4

Also, not passing credentials id should fail `post-kb-credentials-user.test.js`, but it passes. We should change payload it both tests to object that matches actual data that is passed in application
![image](https://user-images.githubusercontent.com/19309423/176382606-64d41da1-cc60-4a08-89c7-857dee1e77f2.png)